swyft/store/store.py

```diff
@@ -420,82 +463,58 @@ def wait_for_simulations(self, indices):
             done = np.all(done)
 
 
-#    # FIXME: Necessary
```

Comment:
[ ] Combine the DirectoryStore and MemoryStore into one class with factory methods

Reply:
We're not going to do this